Makefile

```diff
@@ -129,7 +129,7 @@ requirements-plugins: requirements
 		do AVOCADO_DIRNAME=$(AVOCADO_DIRNAME) make -C $$MAKEFILE requirements &>/dev/null && echo ">> DONE $$MAKEFILE" || echo ">> SKIP $$MAKEFILE";\
 	done
 
-smokecheck:
+smokecheck: clean develop
 	./scripts/avocado run passtest.py
```

Comment:
How about removing the smokecheck completely? It was originally intended for travis check but we decided to use the full `check` instead. Based on the first commit here I guess nobody uses it as it hasn't been working for quite a while...